skydoc/main.py

```diff
@@ -225,10 +227,29 @@ def main(argv):
                                  macro_doc_extractor.description))
 
   if FLAGS.format == "markdown":
+    # convert documentation from markdown that will be in a html table
+    for ruleset in rulesets:
```

Comment:
Might be cleaner to move the logic of iterating through the rules and rendering markdown into `MarkdownWriter` and `HtmlWriter`, respectively.